src/libertem/web/shutdown.py

```diff
@@ -0,0 +1,25 @@
+import tornado
+import logging
+import asyncio
+
+from .state import SharedState
+
+log = logging.getLogger(__name__)
+
+
+class ShutdownHandler(tornado.web.RequestHandler):
+    def initialize(self, state: SharedState, event_registry):
+        self.state = state
+        self.event_registry = event_registry
+
+    async def delete(self):
+        log.info("Handling shutdown button")
+        if self.state.executor_state.executor is not None:
+            await self.state.executor_state.executor.close()
+            self.state.executor_state.executor = None
+        await self.finish({
+            "status": "ok",
+            "messageType": "SERVER_SHUTDOWN"
+        })
+        await asyncio.sleep(1)
```

Comment:
`finish` doesn't fix the issue. Now the test passes with some warning.

Reply:
Hmm, I'll have a look later. Perhaps we have to use a different method of testing for this, as I would prefer not to have a sleep there.

Reply:
any hint?

Reply:
Sorry, didn't find time yet. Will have a look later today